data-raw/process_data_sets.R

```diff
@@ -169,6 +169,12 @@ mythbusters_yawn <- tibble::tibble(group, yawn) %>%
   select(subj, group, yawn)
 usethis::use_data(mythbusters_yawn, overwrite = TRUE)
 
+# Amazon books data from: https://dasl.datadescription.com/datafile/amazon-books
+amazon_books <- 
```

Comment:
The `hard_paper` variable should be of type `factor` not character.

Reply:
@kaceyjj